src/azure-cli/azure/cli/command_modules/cdn/custom.py

```diff
@@ -71,6 +91,304 @@ def update_endpoint(instance,
     return params
 
 
+# pylint: disable=too-many-return-statements
+def create_condition(match_variable=None, operator=None, match_values=None,
+                     selector=None, negate_condition=None, transform=None):
+    if match_variable == 'RemoteAddress':
+        return DeliveryRuleRemoteAddressCondition(
+            parameters=RemoteAddressMatchConditionParameters(
+                operator=operator,
+                match_values=match_values.split(","),
```

Comment:
please use `nargs='+'` instead of splitting it by yourself.

Reply:
fixed